src/components/TradeAssetSearch/hooks/useGetCustomTokenPriceQuery.tsx

```diff
@@ -0,0 +1,80 @@
+import { type AssetId, fromAssetId } from '@shapeshiftoss/caip'
+import { isEvmChainId } from '@shapeshiftoss/chain-adapters'
+import { useQueries, type UseQueryResult } from '@tanstack/react-query'
+import axios from 'axios'
+import { mergeQueryOutputs } from 'react-queries/helpers'
+import { useFeatureFlag } from 'hooks/useFeatureFlag/useFeatureFlag'
+
+type UseGetCustomTokenPricesQueryProps = {
+  assetIds: AssetId[]
+}
+
+type UseGetCustomTokenPricesQueryReturn = {
+  assetId: AssetId
+  marketData: ZerionMarketData | undefined
+}[]
+
+const ZERION_BASE_URL = ' https://zerion.shapeshift.com/'
```

Comment:
```suggestion
const ZERION_BASE_URL = 'https://zerion.shapeshift.com/'
```

Reply:
Actioned in https://github.com/shapeshift/web/pull/7370/commits/45d970641b800d671d18b0b40d9f616e9afe0f6d